core/Makefile

```diff
@@ -164,7 +172,13 @@ $(eval $(call BUILD, \
 	CXX, \
 	%.pb.o, \
 	%.pb.cc $(DEPDIR)/$$@.d, \
-	$$(CXX) -o $$@ -c $$< $$(PROTOCXXFLAGS) $$(DEPFLAGS)))
+	$$(CXX) -o $$@ -c $$< $$(CXXFLAGS) $$(PERMISSIVE) $$(DEPFLAGS)))
+
+$(eval $(call BUILD, \
+	CXX, \
+	bessctl.o, \
```

Comment:
Why does this file require special treatment? If the issue is not straightforward to fix, please leave a brief comment in the Makefile.